src/components/moj-page-header-actions/template.njk

```diff
@@ -0,0 +1,18 @@
+{%- from "moj-menu/macro.njk" import mojMenu %}
+
+<div class="moj-page-header-actions {{- ' ' + params.classes if params.classes}}">
+
+  <div class="moj-page-header-actions__title">
+    {{ params.title.html | safe }}
```

Comment:
I don't think it should accept text. It has to accept a heading (h1, h2, h3).

If we allow it to accept text, then it will just be an unstyled, broken text node (not inside a heading element).

Reply:
Should we require a user to pass in heading html when the macro could be opinionated with a default heading level and class? The html for a heading may therefore only be needed for things like caption html. If we draw an indirect comparison with the input component, isPageHeading changes the nature of the label.

For example, the `title` param would be an object:
```
title: {
  text: "Something interesting"
  headingLevel: "2"
  classes: "govuk-heading-l"
}
```
_`headingLevel` and `classes` would be defaulted to "1" and "govuk-heading-xl" or similar._